static/app/views/issueDetails/streamline/hooks/useFlagSeries.tsx

```diff
@@ -57,6 +57,13 @@ export default function useFlagSeries({query = {}, event}: FlagSeriesProps) {
         const time = getFormattedDate(data.xAxis, 'MMM D, YYYY LT z', {
           local: !selection.datetime.utc,
         });
+
+        const eventIsBefore = moment(event?.dateCreated).isBefore(moment(time));
+        const formattedDate = moment(time).from(event?.dateCreated, true);
+        const suffix = eventIsBefore
```

Comment:
the variable is `suffix`... i guess it makes sense still as it's the suffix of the `.tooltip-footer` element.

Reply:
yup! it's still a suffix that shows up after the time